lib/ElementHandle.js

```diff
@@ -103,6 +103,26 @@ class ElementHandle {
     const objectId = this._remoteObject.objectId;
     return this._client.send('DOM.setFileInputFiles', { objectId, files });
   }
+
+  /**
+   * @return {!Promise<Object>}
+   */
+  async boundingBox() {
+    const box = await this.evaluate(element => {
+      if (!element.ownerDocument.contains(element))
+        return null;
+      const rect = element.getBoundingClientRect();
```

Comment:
UPDATE: I tried again with DOM.getBoxModel and the content quad returned is the position of the box relative to the main frame. I will update once both tests are running (scroll on main frame and nested in iframe)